environment/src/main/java/com/sequenceiq/environment/proxy/domain/ProxyConfig.java

```diff
@@ -200,4 +200,23 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id);
     }
+
+    @Override
+    public String toString() {
+        return "ProxyConfig{" +
+                "id=" + id +
+                ", name='" + name + '\'' +
+                ", description='" + description + '\'' +
+                ", serverHost='" + serverHost + '\'' +
+                ", serverPort=" + serverPort +
+                ", protocol='" + protocol + '\'' +
+                ", userName=" + userName +
```

Comment:
This is a `Secret`.  While it does not reveal anything sensitive, I wonder if it is worth including this in the output.